test/components/plots/clusteredBarPlotTests.ts

```diff
@@ -80,7 +80,7 @@ describe("Plots", () => {
 
       // check that clustering is correct
       var innerScale = (<any>renderer)._makeInnerScale();
-      var off = innerScale.scale("_0");
+      var off = innerScale.scale("0");
       var width = xScale.rangeBand() / 2;
       assert.closeTo(TestMethods.numAttr(bar0, "x") + TestMethods.numAttr(bar0, "width") / 2, xScale.scale(bar0X) - width + off, 0.01
```

Comment:
We currently need to depend on the inner scale in order to determine the offsets for each bar.  While this leads to a less ideal testing infrastructure, we can fix the infrastructure later when we do an audit.